companion/src/logsdialog.cpp

```diff
@@ -19,7 +19,7 @@ logsDialog::logsDialog(QWidget *parent) :
   this->setWindowIcon(CompanionIcon("logs.png"));
   palette.clear();
```

Comment:
Since you have defined new colors for grapsh, remove pallete member from logsDialog class.